pfcpiface/parse-pdr_test.go

```diff
@@ -531,3 +531,72 @@ func Test_portRange_Width(t *testing.T) {
 		)
 	}
 }
+
+func Test_pdr_parsePDR_success(t *testing.T) {
```

Comment:
Not sure on the naming, but I'd like to retain the ability to automatically jump between test and implementation.
(put cursor on function, <kbd>cmd</kbd> + <kbd>shift</kbd> + <kbd>T</kbd>)

Reply:
Didn't know about this `cmd` + `shift` + `T` feature. I think it's a great and we should definitely be able to use it.

I don't remember if we agreed to have positive and negative tests so I'll just list some combinations:

1. `Test_parsePDR()`
2. `TestParsePDR()`
3. `TestParsePDRSuccess()`
4. `TestParsePDRShouldError()`
5. `TestParsePDRFail()`
6. `Test_parsePDR_success()` (not a big fan of using an hybrid between camel and snake case, on go) 
7. `Test_parsePDR_error()`
8. `Test_parsePDR_should_error()`
9. `Test_parsePDR_fail()`

If you have more, please go ahead and list them. (I'd go for `2` and `4`, in case we agreed on having separate positive and negative tests)

Reply:
I like 3 and 5 :)

Reply:
Starting with `Test` is mandatory for `go test` to pick it up.
I think I have identified the following pattern from the Goland IDE that avoids collisions:

- Receivers get added to the name with a trailing dash, because multiple types can implement `String()`
- Unexported types/functions are lowercase and have a dash in front

```go
type PublicStruct struct {}
func (f PublicStruct) PublicFn() {}
func (f PublicStruct) privateFn() {}

type privateStruct struct {}
func (p privateStruct) PublicFn() {}
func (p privateStruct) privateFn() {}
```

```go
func TestPublicStruct_PublicFn(t *testing.T) {}
func TestPublicStruct_privateFn(t *testing.T) {}

func Test_privateStruct_PublicFn(t *testing.T) {}
func Test_privateStruct_privateFn(t *testing.T) {}
```

In my testing the IDE only picked up that style correctly. Make sure to navigate the cursor to a different location before testing the jump feature.
Appending anything with a leading underscore seems be ok too:

![image](https://user-images.githubusercontent.com/7455710/153084167-e3b60c6a-5457-4348-8fce-9932d03c5c36.png)

Reply:
Got it - I'm fine with everything what makes our life easier, so let's follow the style that is compatible with IDE.

Reply:
mhh, I feel we need more experience before setting up a rule on naming. For now, I'd say everything goes and we clean it up once we know more.